src/components/TodoList/TodoList.tsx

```diff
@@ -1,224 +1,101 @@
 /* eslint-disable */
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+import { Loader } from '../Loader';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { setCurrentTodo } from '../../features/currentTodo';
+import { getPrepapredTodos } from '../../services/getPreparedTodos';
```

Comment:
There is a typo in the import statement. It should be 'getPreparedTodos' instead of 'getPrepapredTodos'. This will cause an error when trying to use the function.